src/main.zig

```diff
@@ -119,12 +122,79 @@ fn rocRepl(allocator: Allocator, opt: RocOpt, args: []const []const u8) !void {
     fatal("not implemented", .{});
 }
 
-fn rocFormat(allocator: Allocator, opt: RocOpt, args: []const []const u8) !void {
-    _ = allocator;
+fn rocFormat(allocator: Allocator, args: []const []const u8) !void {
+    const path = if (args.len > 0) args[0] else "main.roc";
```

Comment:
Should we rename this to `roc_file_path`? I love to be able to know what a variable holds without having to look around.